src/M6Web/Bundle/DaemonBundle/Command/DaemonCommand.php

```diff
@@ -209,20 +205,17 @@ public function run(InputInterface $input, OutputInterface $output)
         // Force the creation of the synopsis before the merge with the app definition
         $this->getSynopsis();
 
-        // Add the signal handler
-        if (function_exists('pcntl_signal')) {
```

Comment:
I removed it because there is already `"ext-pcntl" : "*"`, in the composer.json.